src/server/models/list.ts

```diff
@@ -200,6 +215,49 @@ export async function updateList(
   }
 }
 
+export async function removeUserFromList(listId: number, userEmail: string): Promise<void> {
+  try {
+    await prisma.list.update({
+      where: {
+        id: listId,
+      },
+      data: {
+        users: {
+          disconnect: {
+            email: userEmail,
+          },
+        },
+      },
+    });
+    logger.info(`User ${userEmail} disconnected from list ${listId}`);
+  } catch (error) {
+    logger.error(`removeUserFromList Error: ${(error as Error).message}`);
+    throw error;
+  }
+}
+
+async function createUsersFromEmails(email: string): Promise<void> {
+  await prisma.user.create({
```

Comment:
Fine for now, but generally it's useful to return created entities so you can use it without doing another lookup.